src/scene_battle_rpg2k3.cpp

```diff
@@ -1312,16 +1352,25 @@ void Scene_Battle_Rpg2k3::Escape(bool force_allow) {
 	if (force_allow || TryEscape()) {
 		// There is no success text for escape in 2k3
 		Game_System::SePlay(Game_System::GetSystemSE(Game_System::SFX_Escape));
+		SetWait(10, 30);
 		EndBattle(BattleResult::Escape);
```

Comment:
`EndBattle()` pops the scene, so this wait never actually gets exercised.

Reply:
Fixed.